app/controllers/api/v1/games_controller.rb

```diff
@@ -1,6 +1,6 @@
 module Api
   module V1
-    class GamesController < ActionController::API
```

Comment:
I'm not sure why taking the namespacing away from the parent would make a difference or why they have it there in the first place.